errors/error.go

```diff
@@ -55,6 +55,16 @@ func CheckErr(err error, code int, optMsg string) {
 			log.Print("DELETE_FILE_ERROR", "[", code, "]: ", err, ". ", optMsg)
 		case 300:
 			log.Fatal("APPLICATION_ERROR", "[", code, "]: ", err, ". ", optMsg)
+		case 400:
+			log.Fatal("REPOSITORY_DOWNLOAD_ERROR", "[", code, "]: ", err, ". ", optMsg)
+		case 401:
+			log.Fatal("CREATE_ZIP_FILE_ERROR", "[", code, "]: ", err, ". ", optMsg)
+		case 402:
+			log.Fatal("ZIP_FILE_READ_ERROR", "[", code, "]: ", err, ". ", optMsg)
```

Comment:
Might be nice to be consistent here and have `READ_ZIP_FILE_ERROR` to go with `CREATE_ZIP_FILE_ERROR`?

Reply:
Good spot.